app/models/direct_file_data.rb

```diff
@@ -502,14 +504,33 @@ def fed_w2_state=(value)
     write_df_xml_value(__method__, value)
   end
 
+  def primary_blind
+    create_or_destroy_df_xml_node(__method__, true, 'VirtualCurAcquiredDurTYInd')
+    write_df_xml_value(__method__, "X")
+  end
+
+  def spouse_blind
+    create_or_destroy_df_xml_node(__method__, true, 'VirtualCurAcquiredDurTYInd')
+    write_df_xml_value(__method__, "X")
+  end
+
+  def is_primary_blind?
+    return false unless parsed_xml.at('PrimaryBlindInd')
+    true
```

Comment:
[dust/pebble] why this style over just `return parsed_xml.at('PrimaryBlindInd')`. I find the "false unless" logic pretty head-bending to read if we can avoid it

Reply:
Yeah, thanks for the callout there, will fix. Had more complex logic originally that I thought warranted the guard statement but it's confusing now.